src/apdu_pubkey.c

```diff
@@ -53,19 +53,10 @@ static void bip32_path_to_string(char *const out, size_t const out_size, apdu_pu
 }
 
 void render_pkh(char *const out, size_t const out_size,
-                render_address_payload_t const *const payload) {
+                render_address_payload_t const *const payload, size_t payload_len) {
     uint8_t base32_buf[256];
     size_t base32_len = 0;
-    size_t payload_len = 0;
-    bool is_bech32m = 0;
-    if (payload->full_version.address_format_type == ADDRESS_FORMAT_TYPE_FULL_VERSION) {
-        payload_len = sizeof(payload->full_version);
-        is_bech32m = 1;
-    } else if (payload->short_version.address_format_type == ADDRESS_FORMAT_TYPE_SHORT) {
-        payload_len = sizeof(payload->short_version);
-    } else {
-        payload_len = sizeof(payload->code_hash_data_or_type);
-    }
+    bool is_bech32m = 1;
```

Comment:
Why do you keep a boolean variable with only hardcoded `true` value?

Reply:
This is due to the deprecation of the bech32 (short address format), which now only uses [bech32m](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0021-ckb-address-format/0021-ckb-address-format.md#full-payload-format) to encode a script to an address. All test cases under `/tests` were encoded with bech32m.

Reply:
Ok, but this variable could then be removed, and a `1` or `true` value can be passed to the function `bech32_encode`